terraso_backend/apps/soil_id/graphql/soil_id.py

```diff
@@ -118,15 +116,19 @@ class DataBasedSoilMatches(graphene.ObjectType):
     matches = graphene.List(graphene.NonNull(DataBasedSoilMatch), required=True)
 
 
+class LabColorInput(graphene.InputObjectType):
```

Comment:
Suggestion: Does "Lab" stand for something? If so, maybe make a comment or name them more clearly?

Reply:
nope, it's just LAB :shrug:  https://en.wikipedia.org/wiki/CIELAB_color_space

Reply:
suggestion: call this `LABColorInput`, since we capitalized `RGB` elsewhere.